erasure-readfile.go

```diff
@@ -25,6 +25,60 @@ import (
 	"github.com/klauspost/reedsolomon"
 )
 
+// isSuccessDecodeBlocks - do we have all the blocks to be successfully decoded?.
```

Comment:
Would it help to add a note saying this function can be applied only on an ordered slice of disks?